status/statusPage.html

```diff
@@ -0,0 +1,699 @@
+                    callback(rawFile.responseText);
+                } else {
+                    callback('{}');
+                }
+            };
+            rawFile.send(null);
+        }
+
+        function getDetails(object) {
+            let title = "";
+            title += "Id: " + object['_id'] + "<br>";
+            title += "Gem5 Binary: " + object['gem5_binary'] + "<br>";
+            title += "Run Script: " + object['run_script'] + "<br>";
+            title += "Gem5 Artifact: " + object['gem5_artifact'] + "<br>";
+            title += "Gem5 Git Artifact: " + object['gem5_git_artifact'] + "<br>";
+            title += "Run Script Git Artifact: " + object['run_script_git_artifact'] + "<br>";
+            title += "Timeout: " + object['timeout'] + "<br>";
+            title += "Gem5_name: " + object['gem5_name'] + "<br>";
+            title += "Script_name: " + object['script_name'] + "<br>";
+            title += "Running: " + object['running'] + "<br>";
+            title += "Enqueue_time: " + object['enqueue_time'] + "<br>";
+            title += "Start_time: " + object['start_time'] + "<br>";
+            title += "End_time: " + object['end_time'] + "<br>";
+            title += "Return_code: " + object['return_code'] + "<br>";
+            title += "Kill_reason: " + object['kill_reason'] + "<br>";
+            title += "PId: " + object['pid'] + "<br>";
+            title += "Task_id: " + object['task_id'] + "<br>";
+            title += "Results: " + object['results'] + "<br>";
+            title += "Linux_binary: " + object['linux_binary'] + "<br>";
+            title += "Disk_image: " + object['disk_image'] + "<br>";
+            title += "Linux_binary_artifact: " + object['linux_binary_artifact'] + "<br>";
+            title += "Disk_image_artifact: " + object['linux_binary_artifact'] + "<br>";
+            title += "Disk_name: " + object['disk_name'] + "<br>";
+            title += "String: " + object['string'] + "<br>";
+            title += "Hash: " + object['hash'] + "<br>";
+            title += "Type: " + object['type'] + " <br>";
+            title += "Current_time: " + object['current_time'];
+            return title;
+        }
+
+        //Runs through the JSON file, at each object, it creates an ID based on the linux name and parameters. It searches to see if the ID exists,
+        //if it does, it assigns the box with the ID the correct color and hovertext.
+        function determineColor() {
+            readTextFile(JSONPath, function (json) {
+                let parsedJSON = JSON.parse(json);
+                for (let i in parsedJSON) {
+                    let idToTest = parsedJSON[i]['linux_name'] + " " + parsedJSON[i]['params'][0] + " " + parsedJSON[i]['params'][1] + " " + parsedJSON[i]['params'][2] + " " + parsedJSON[i]['params'][3];
+                    if (document.getElementById(idToTest)) {
+                        document.getElementById(idToTest).onmouseover = function () {
```

Comment:
Can we make this onclick instead? I'm having trouble with the center blocks in a grid. I keep mousing over other blocks when moving my cursor :). In fact, it might make sense for the blocks to be links. Or, not, I could be wrong ;)